### PR TITLE
Added few tests on atomic host

### PR DESCRIPTION
**Pull request by sghai**
A few basic tests around atomic hosts for now. Will add more once I get better idea what will be in next release.

### CODE REVIEW

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -412,3 +412,95 @@ def test_positive_update_with_user(self):
...
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be updated with new content-view
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_execute_cmd_on_atomic_host_with_job_templates(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_host.py]: maybe `def test_positive_execute_cmd_with_job_templates(self):`?

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -412,3 +412,95 @@ def test_positive_update_with_user(self):
...
+
+        @Assert: Atomic host should be registered successfully and listed under
+        content-hosts/Hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_register_pre_installed_atomic_host_using_ak(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_host.py]: maybe `def test_positive_register_using_ak`?

**Comment by oshtaier** [comment]
ACK pending non-blocking comments

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (sghai) [inline tests/foreman/ui/test_host.py]: Yeah.. I was thinking to use same names. But we may have host (content-host) specific tests which are not related to `atomic_host`. So I added `atomic_host` in test names just to differentiate between content-host and atomic.

**Comment by sghai** [comment]
@oshtaier : I'm using `atomic_host` in test names just to differentiate between `content-host` and `atomic host`. We can use the suggested names if we move all atomic_host tests to separate test_module.

**Comment by sghai** [comment]
@all: comments please. :)

File: tests/foreman/ui/test_host.py [hunk 3]
```diff
@@ -412,3 +412,95 @@ def test_positive_update_with_user(self):
 
         @Status: Manual
         """
+
+    @stubbed()
+    @tier3
+    def test_positive_provision_atomic_host(self):
+        """Provision an atomic host on libvirt and register it with server
```
Comment (sthirugn) [inline tests/foreman/ui/test_host.py]: s/server/satellite?

**Comment by sthirugn** [comment]
I am good with @sghai's explanation of using `atomic-host` in the test case names to differentiate these tests from the content host tests.

**Comment by sthirugn** [comment]
ACK pending 1 minor comment.

**Comment by sghai** [comment]
Comment addressed. Please review and merge. thanks

**Comment by sthirugn** [comment]
ACK

**Comment by sthirugn** [comment]
There is an other non blocking comment, so merging now.

**Pull request merged**